index.html

```diff
@@ -1236,34 +1389,66 @@ <h5>Duration</h5>
+							base direction is right-to-left).</p>
 
-						<p class="note">It is important to differentiate the language <em>of the publication</em> from
-							the language and the base direction of the individual resources that compose it. If such
-							resources are, for example, in HTML, the language and direction need to be set in those
-							resources, too. The language and base direction of the publication are not inherited.</p>
-
-						<p>Similarly, each natural language property value in the manifest (e.g., <a href="#pub-title"
-								>title</a>, <a href="#creators">creators</a>) is a <a href="#value-localizable-string"
-								>localizable string</a>.</p>
+						<table class="zebra">
+							<thead>
+								<tr>
+									<th>Term</th>
+									<th>Description</th>
+									<th>Required Value</th>
+									<th>Value Category</th>
+									<th>[[!schema.org]] Mapping</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr>
+									<td>
+										<code data-dfn-for="PublicationManifest">
+											<dfn>inLanguage</dfn>
+										</code>
+									</td>
+									<td>Default <a>language</a> for the publication as well as the textual manifest
+										values</td>
+									<td>One or more language code as defined in&#160;[[!bcp47]]</td>
+									<td>
+										<a href="#value-array">Array</a> of <a href="#value-literal">Literals</a>
+									</td>
+									<td>
+										<a href="https://schema.org/inLanguage"><code>inLanguage</code></a> (<a
+											href="https://schema.org/Property">Property</a>) </td>
+								</tr>
+								<tr>
+									<td>
+										<code data-dfn-for="PublicationManifest">
+											<dfn>inDirection</dfn>
+										</code>
+									</td>
+									<td>Default <a>base direction</a> for the publication as well as the textual
+										manifest values</td>
+									<td><code>ltr</code>, <code>rtl</code>, or <code>auto</code></td>
+									<td>
+										<a href="#value-literal">Literal</a>
+									</td>
+									<td>(None)</td>
+								</tr>
+							</tbody>
+						</table>
 
-						<p class="note">For more information about localized strings on the Web, refer to
-							[[string-meta]].</p>
+						<p>The natural language MUST be a well-formed in terms of &#160;[[!bcp47]], while the base
+							language direction MUST have one of the following values:</p>
 
-						<p>The natural language and base direction can be set for both the <a
-								href="#manifest-default-language-and-dir">publication</a> and the <a
-								href="#manifest-specific-language-and-dir">natural language properties values</a> of the
-							manifest.</p>
+						<ul data-include="common/html/dir-values.html" data-include-replace="true"></ul>
 
 						<p>If a user agent requires the language and one is not available in the manifest (either
 							globally or specifically for that property), or the obtained value is invalid, the user
```

Comment:
Probably worth emphasizing 'publication language', rather than just 'language'

Also, when referring to "the non-empty language" it is better to refer explicitly to 2.5.1